### DISCUSSION TITLE
Can I prevent the HTML picture element from being erased?

### DISCUSSION THREAD

**Discussion by PetersOtto** [datenstrom/community#705] [Ask a question] [answered]
Hello all!
I am just playing a bit around with the image extension.
It would be great to have a solution for using the picture-tag.

I searched, but I do not found an answer where the picture-Tag will be erase. Is it possible to prevent yellow from erasing this tag?

Best regards

**Comment by schulle4u** [comment, marked as answer]
Currently Yellow doesn't support the `<picture`> element. The list of allowed HTML tags and attributes is maintained in the [Core extension](https://github.com/datenstrom/yellow-extensions/tree/master/source/core). Open the file `core.php` and search for the `normaliseData` function around line 3395. You can add missing tags to the `$elementsHtml` array, however this will be overwritten with the next extension update. Therefore don't hesitate to [publish your modifications](https://github.com/datenstrom/yellow-extensions/tree/master/source/publish) and conclude with a pull request. 

Let us know if you need further help.

**Reply by PetersOtto** [reply]
Thank you very much! 

In my opinion it is not important to offer images in different sizes, when you use a normal blog with a width from around 1000px. 

However when you plan to use full screen images, you have to provide it.

When I see how good your code looks, I do not dare to show my code :) 

Have a nice weekend!

**Reply by markseuffert** [reply]
I didn't see a pull request. Added the [picture](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element to the HTML filter, now available in latest version.
